bazel/coverage/collect_cc_coverage.sh

```diff
@@ -61,7 +61,15 @@ function init_gcov() {
   # we would need to invoke it with "llvm-cov gcov").
   # For more details see https://llvm.org/docs/CommandGuide/llvm-cov.html.
   GCOV="${COVERAGE_DIR}/gcov"
-  ln -s "${COVERAGE_GCOV_PATH}" "${GCOV}"
+  if [ ! -f "${COVERAGE_GCOV_PATH}" ]; then
+    echo "GCov does not exist at the given path: '${COVERAGE_GCOV_PATH}'"
+    exit 1
```

Comment:
unfortunately exit 1 does nothing in this script, even if you exit 1 at the beginning of this script bazel won't treat it as test error because [the script evaluating](https://github.com/bazelbuild/bazel/blob/master/tools/test/collect_coverage.sh#L150) this one doesn't have `set -e`.

Reply:
I see. What should we do in this case? I still prefer this because it clearly logs the error and bazel will fail later (since the gcov won't run).

Reply:
Let's just have -e in this script, so `ln` automatically fails the script?

Reply:
If you `set -e` at beginning of the script should fail in `ln`?

I'm trying to keep this file as close as possible to upstream collect_cc_coverage.sh until we make it upstream to Bazel.

There's no way to propagate this up with `eval` which in `collect_coverage.sh`.

Reply:
Hi @lizan, sorry for the back and forth, but I would like to clarify once more before I change it to just `set -e`.

This PR is copying the script from upstream bazel: https://github.com/bazelbuild/bazel/blob/d7432e2096f3f1ea23587da6f3ef2fb5d5a557a9/tools/test/collect_cc_coverage.sh#L58

Shouldn't we keep the scripts in sync as you mention?